assets/section-footer.css

```diff
@@ -479,11 +479,30 @@ noscript .localization-selector.link {
     text-align: center;
   }
 }
+.footer-block-brand-information {
+  text-align: center;
+}
 
-.footer-block-image > img {
+.footer-block-image > img,
+.footer-block-brand-information > img {
   height: auto;
 }
 
+.footer-block .footer-block-brand-information .footer__list-social.list-social {
+  justify-content: center;
+}
+
+@media screen and (min-width: 750px) {
+  .footer-block:not(:only-child) .footer-block-brand-information {
+    text-align: left;
+  }
+
+  .footer-block:not(:only-child) .footer-block-brand-information .footer__list-social.list-social {
+    justify-content: flex-start;
+    margin-left: -1.3rem;
+  }
+}
+
 .footer-block__details-content .placeholder-svg {
   max-width: 20rem;
 }
```

Comment:
Not related to any of your changes on the PR but the default column gap value looks tight on those footer columns: 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/02-51-q498i-puil5.png)
</details>